frb/tests/test_associate.py

```diff
@@ -0,0 +1,111 @@
+import os
+import numpy as np
+from pkg_resources import resource_filename
+
+import pandas
+
+from astropy import units
+
+from frb.associate import frbassociate
+
+import pytest
+
+remote_data = pytest.mark.skipif(os.getenv('FRB_GDB') is None,
+                                 reason='test requires dev suite')
+
+@remote_data
+def test_individual():
+    from frb.associate import frbs
+    config = getattr(frbs, 'FRB180924'.lower())
+    frbA = frbassociate.run_individual(config)
+    # Test
+    assert isinstance(frbA.candidates, pandas.DataFrame)
+
+
+@remote_data
+def test_step_by_step():
+    from frb import frb
+
+    # Testing
+
+    # #########################
+    # Prep
+    frb180924 = frb.FRB.by_name('FRB180924')
+
+    # Instantiate
+    max_radius = 10.  # in arcseconds
+    frbA_180924 = frbassociate.FRBAssociate(frb180924,
+                                            image_file=resource_filename('frb', 'associate/dev/FRB180924_DESr.fits'),
+                                            max_radius=max_radius)
+
+    # Threshold
+    frbA_180924.threshold()
+
+    # Segment
+    frbA_180924.segment()#show=True, outfile='dev/FRB180924_segm.png')
+
+    # Photometry
+    frbA_180924.photometry('MAGZERO', 'r')#, show=True, outfile='dev/FRB180924_aper.png')
+
+    # Candidates
+    plate_scale = frbA_180924.header['CD2_2'] * 3600. * units.arcsec  # arcsec
```

Comment:
Better to compute CDELT1 or CDELT2 and use those as plate scales just in case the image is rotated.

CDELT2 = rms(CD[2,1], CD[2,2]).
Usually CDELTS are equal but I think they're different in the case of KCWI.

Reply:
this is just a test